DESCRIPTION

```diff
@@ -1,6 +1,6 @@
 Package: repro
 Type: Package
-Title: Easy Setup of a Reproducible Workflow
+Title: Automated Setup of a Reproducible Workflow
```

Comment:
I think the phrase "A reproducible workflow" creates the impression that it's not as customizable as it really is. I would say something like: Automated setup of solutions/components/dependencies/? for reproducible workflow**S**

Reply:
True, do you like "Automated Setup of a Reproducible Workflows and their components"?
I don't fancy the "of" chain.

Reply:
Automated setup of reproducible workflows and (their dependencies|required tools)?